datasets/alt/alt.py

```diff
@@ -0,0 +1,429 @@
+    "alt-my": "https://www2.nict.go.jp/astrec-att/member/mutiyama/ALT/my-alt-190530.zip",
+    "alt-my-transliteration": "https://www2.nict.go.jp/astrec-att/member/mutiyama/ALT/my-en-transliteration.zip",
+    "alt-my-west-transliteration": "https://www2.nict.go.jp/astrec-att/member/mutiyama/ALT/western-myanmar-transliteration.zip",
+    "alt-km": "https://www2.nict.go.jp/astrec-att/member/mutiyama/ALT/km-nova-181101.zip",
+}
+
+_SPLIT = {
+    "train": "https://www2.nict.go.jp/astrec-att/member/mutiyama/ALT/URL-train.txt",
+    "dev": "https://www2.nict.go.jp/astrec-att/member/mutiyama/ALT/URL-dev.txt",
+    "test": "https://www2.nict.go.jp/astrec-att/member/mutiyama/ALT/URL-test.txt",
+}
+
+_WIKI_URL = "https://www2.nict.go.jp/astrec-att/member/mutiyama/ALT/ALT-Parallel-Corpus-20191206/URL.txt"
+
+
+class AltParallelConfig(datasets.BuilderConfig):
+    """BuilderConfig for ALT."""
+
+    def __init__(self, languages, **kwargs):
+        """BuilderConfig for ALT.
+
+        Args:
+            for the `datasets.features.text.TextEncoder` used for the features feature.
+
+            languages: languages that will be used for translation. it should be one of the
+            **kwargs: keyword arguments forwarded to super.
+        """
+
+        name = "alt-parallel"
+
+        description = "ALT Parallel Corpus"
+        super(AltParallelConfig, self).__init__(
+            name=name,
+            description=description,
+            version=datasets.Version("1.0.0", ""),
+            **kwargs,
+        )
+
+        available_langs = set(
+            ["bg", "en", "en_tok", "fil", "hi", "id", "ja", "khm", "lo", "ms", "my", "th", "vi", "zh"]
+        )
+        for l in languages:
+            assert l in available_langs
+
+        self.languages = languages
+
+
+class Alt(datasets.GeneratorBasedBuilder):
+    """Asian Language Treebank (ALT) Project"""
+
+    BUILDER_CONFIGS = [
+        AltParallelConfig(
+            languages=["bg", "en", "en_tok", "fil", "hi", "id", "ja", "khm", "lo", "ms", "my", "th", "vi", "zh"]
+        ),
+        datasets.BuilderConfig(name="alt-en", version=datasets.Version("1.0.0"), description="English ALT"),
+        datasets.BuilderConfig(name="alt-jp", version=datasets.Version("1.0.0"), description="Japanese ALT"),
+        datasets.BuilderConfig(name="alt-my", version=datasets.Version("1.0.0"), description="Myanmar ALT"),
+        datasets.BuilderConfig(name="alt-km", version=datasets.Version("1.0.0"), description="Khmer ALT"),
+        datasets.BuilderConfig(
+            name="alt-my-transliteration",
+            version=datasets.Version("1.0.0"),
+            description="Myanmar-English Transliteration Dataset",
+        ),
+        datasets.BuilderConfig(
+            name="alt-my-west-transliteration",
+            version=datasets.Version("1.0.0"),
+            description="Latin-Myanmar Transliteration Dataset",
+        ),
+    ]
+
+    DEFAULT_CONFIG_NAME = "alt"
```

Comment:
```suggestion
    DEFAULT_CONFIG_NAME = "alt-parallel"
```

fix the default config name